src/com/Jacksonnn/DCCore/Rankup/Rankup.java

```diff
@@ -87,7 +87,7 @@ public static void attemptRankup(Player player, String oldGroup, String newGroup
                 ", on ranking up from " + oldGroup + " to " + newGroup + "!");
     }
 
-    private static boolean isPlayerInGroup(Player player, String group) {
+    public static boolean isPlayerInGroup(Player player, String group) {
```

Comment:
Allows me to call this in GuestQuizListener to properly determine the player's rank